Makefile

```diff
@@ -5,7 +5,7 @@ ifdef TTY
 	AWS_CREDS_MOUNT = --mount type=bind,source=$(HOME)/.aws/credentials,destination=/root/.aws/credentials,readonly
 endif
 
-DEPLOYER_IMAGE = quay.io/powerhome/deployer:master-c94bf553840b07335fbb8904d5a9963dd5ffce00-336
+DEPLOYER_IMAGE = image-registry.powerapp.cloud/app/deployer@sha256:349dfeda19a069bfe1167f7369acbd91f313bc459cd73ed6082a868c2a3e626b
```

Comment:
Why not continue to use an image tag that allows us to compare with the APP repo?